### DISCUSSION TITLE
add reversed device_class (eg Safety 'on' =  Safe) to binary_sensor

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#22030] [Dashboard]
not sure how to best describe this, but what we now have are all 'incidents' on the device_class name.

problem on means problem,
moisture on means leakage
motion on means motion

etcetc

however, Looking for a device_class to indicate a positive safe situation in 'on', I checked `device_class: safety`, because the name implies that is the correct class for safety.

However, from the mentioned perspective above it is doing the exact opposite thing:
Safety 'on' means unsafe. Safety 'off means safe

<img width="861" alt="Scherm­afbeelding 2024-09-19 om 08 11 37" src="https://github.com/user-attachments/assets/ba7a6a01-5cc0-44db-839e-cfaa1b75542b">

In other words, we cant use this, or any other binary_sensor device_class to have a 'safe' default situation, and have it toggle on the exception.

hence this FR Discussion, to add a class that has 'on' for state when all is good, Safe, and 'off' in the not safe situation

(yes I did check the theme colors, have them set, but can not change the icon there, which remains using the opposite logic. Also, the involved binaries are complex template entities, and it would become really hard to rewrite those and inverse completely. 

A 'simple' device_class with describe logic would be ever so helpful.
Thanks for considering

**Comment by Mariusthvdb** [comment]
sorry this is for FR in the community as its about backend functionality. I copied it there https://community.home-assistant.io/t/add-reversed-device-class-eg-safety-on-safe-to-binary-sensor/772487
